src/chat/view.py

```
@@ -0,0 +1,100 @@
+
+from src.agent import Agent
+from src.chat.service import (ConversationService, get_agent,
+                              get_conversation_service, query_generator)
+from src.core import Conversation, Message, Role
+
+router = APIRouter(prefix='/conversations')
+
+
+@router.get('')
+async def list_conversations(
+        conversation_service: ConversationService = Depends(get_conversation_service)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)